samples/bot-formatting-cards/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,38 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.14/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.14",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.microsoft.teams.differentformattingcards",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "Different Formatting Cards",
+    "full": "Different Formatting Cards"
+  },
+  "description": {
+    "short": "Different Formatting Cards",
+    "full": "This sample demonstrates the use of different formatting on Cards."
```

Comment:
1. Can we add "Adaptive" before cards? like "This sample demonstrates the use of different formatting on adaptive Cards" and "Different Formatting Adaptive Cards"
2. "This sample demonstrates the use of different formatting on Cards." , "C" should small

Reply:
https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/689/files#diff-824fbd3defb344889effcfecba9c5a1ae303b488e565c47c32494597545bba6b

Reply:
Fixed